create_framework/dependency_injection.rst

```diff
@@ -25,7 +25,7 @@ to it::
             $argumentResolver = new HttpKernel\Controller\ArgumentResolver();
 
             $dispatcher = new EventDispatcher();
-            $dispatcher->addSubscriber(new HttpKernel\EventListener\RouterListener($matcher));
+            $dispatcher->addSubscriber(new HttpKernel\EventListener\RouterListener($matcher, new RequestStack()));
```

Comment:
you should use the same request stack than in the ``parent::__construct`` call. Using an independent stack will be broken